evergreen/evergreen.yml

```diff
@@ -0,0 +1,41 @@
+tasks:
+- name: compile
+  commands:
+    - command: git.get_project
+      params:
+        directory: src
+    - func: "compile envoy"
+- name: test
+  commands:
+    - command: git.get_project
+      params:
+        directory: src
+    - func: "test envoy"
+
+functions:
+  "compile envoy":
+    command: subprocess.exec
+    params:
+      binary: bash
+      args:
+      - "src/evergreen/compile-centos7-evergreen.sh"
+  "test envoy":
+    command: subprocess.exec
+    params:
+      binary: bash
+      args:
+      - "src/evergreen/test-centos7-evergreen.sh"
+
+buildvariants:
+- name: centos7
+  display_name: CentOS 7
+  run_on:
+    - rhel80-xlarge
```

Comment:
Perhaps dumb question, but shouldn't we be using rhel70 images for CentOS 7?

Reply:
This is how SRE repo runs the podman (docker). Since we use containers, it doesn't have to be the same as long as it's x86. I'd hope to keep it the same as the SRE repo. Adding a comment.